### DISCUSSION TITLE
WECC model with wind turbine

### DISCUSSION THREAD

**Discussion by Leonawyciii** [CURENT/andes#451] [Modeling] [answered]
Hi Dr. Cui and other developers,

Now I have PSSE file data with 40 wind turbines, the Renewable machine in PSSE is WT4G1 and the electrical part is WT4E1, but it seems no matching model in Andes. I try to convert the raw file first, then change wind turbine model from WT4G1 to Andes model(similar in ieee14 wtn). But it still shows errors when setup the system. Do you have any suggestions?  or for wecc 179 model, do we have any case data with wind turbines?

**Comment by jinningwang** [comment]
As far as I know, we don't have a ready-to-use WECC case that integrates wind models.
I would suggest you convert the original WECC system which has no wind model. Then, in the converted ANDES xlsx file, replace the synchronous generators with wind models manually.

**Reply by cuihantao** [reply]
We might want to make good use of the existing models.

@Leonawyciii, are you a student at UTK? `WT4G1` is similar to `REGCA1`, and `WT4E1` is similar to `REECA1`. You can first inspect the model diagrams (do a Google search) and identify matching parameters. Then, you can use PSS/E to replace the `WT4*` models with the corresponding `RE*` models, save the dynamic files in `.dyr`, and load it using ANDES.

The `RE*` models is called "second generation renewable models". You will find more interesting reads from WECC and Powerworld by searching on Google.

**Reply by Leonawyciii** [reply]
yes, I am a UTK student and trying to use the Andes to finish the course project. Thank you for the detailed suggestions. I will try to work on that.

**Reply by Leonawyciii** [reply]
when I try to set the model con value for regca1 and reeca1 model, no problem finding the matching parameters, but how to deal with some values that cannot find the matching parameters with wt4g1 /wt4e1 or default parameters？

**Reply by jinningwang** [reply]
For the non-matching parameters between RE* and WT*, andes will use default values in file conversion.
You can modify the parameters in the converted ANDES xlsx file if necessary.


> when I try to set the model con value for regca1 and reeca1 model, no problem finding the matching parameters, but how to deal with some values that cannot find the matching parameters with wt4g1 /wt4e1 or default parameters？

**Reply by cuihantao** [reply, marked as answer]
> For the non-matching parameters between RE* and WT*, andes will use default values in file conversion.

I don't think this was the question.

When manually converting `WT` models from `RE` models, approximations can be unavoidable. You can use your best judgement for the non-matching parameters based on the functionality of the blocks. Some manual tuning may be needed.

**Reply by Leonawyciii** [reply]
Yes, I try to use default values and set others based on some study cases. And now it works in the Andes. thank you!
